x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/TransportEsqlStatsAction.java

```diff
@@ -63,13 +60,7 @@ public TransportEsqlStatsAction(
 
     @Override
     protected DiscoveryNode[] resolveRequest(EsqlStatsRequest request, ClusterState clusterState) {
-        if (featureService.clusterHasFeature(clusterState, ESQL_STATS_FEATURE)) {
-            // use the whole cluster
-            return super.resolveRequest(request, clusterState);
-        } else {
-            // not all nodes in the cluster have upgraded to esql - just use this node for now
-            return new DiscoveryNode[] { clusterService.localNode() };
-        }
+        return super.resolveRequest(request, clusterState);
```

Comment:
Do we even need  the override if we defer to super?

Reply:
Er, nope. sorry. Was being too quick on the refactor.